src/pages/cart/Cart.jsx

```diff
@@ -1,8 +1,94 @@
-import "./cart.module.css";
+import "./cart.css";
+import { Link } from "react-router-dom";
 import { useDocumentTitle } from "hooks";
+import { shoppingImage } from "assets/svgs";
+import { useAuth } from "context/auth-context";
 const Cart = () => {
   useDocumentTitle("Cart");
-  return <>Cart</>;
+  const {
+    authState: { cart },
+  } = useAuth();
+  return cart.length === 0 ? (
+    <div style={{ textAlign: "center" }}>
```

Comment:
Inline css is not good i think

Reply:
Will look into that, When I do code refactor Thank you :)